### PR TITLE
Docs: Update/Refresh OneLogin SSO guide

### PR DESCRIPTION
**Pull request by alexfornuto**
Notable updates:

- Uses `tctl sso` to create/test connector
- fixes webapi paths
- Updates the `samlauthentication` partial to fit under the "create a connector" section of docs, instead of being a bullet point for prerequisites. This way, the reader isn't setting the default auth type before the connector is created.

**Note:** Backports beyond 13 need to remove the `tctl sso` command.

### CODE REVIEW

File: examples/resources/onelogin-connector.yaml [hunk 1]
```diff
@@ -3,14 +3,24 @@ version: v2
...
-    - {name: "groups", value: "dev", roles: ["access"]}
+  - name: groups
+    roles:
+    - editor
+    value: admin
+  - name: groups
+    roles:
+    - access
+    value: dev
+  audience: https://tp.alexf.xyz:443/v1/webapi/saml/acs/onelogin
```
Comment (zmb3) [inline examples/resources/onelogin-connector.yaml:16]:
```suggestion
  audience: https://<cluster-url>/v1/webapi/saml/acs/onelogin
```

We should be consistent in this file - there's a generic placeholder "variable" in one place, a specific alexf.xyz domain in another place, and a teleport.example.com in another place.

Also, `<cluster-url>` is probably not the right name anyway, as it's not a URL that goes there. Maybe `<proxy-hostname>` instead?

File: docs/pages/access-controls/sso/one-login.mdx [hunk 2]
```diff
@@ -1,52 +1,59 @@
...
+From the application's **Configuration** page, set the following values:
 
-`https://teleport.example.com:3080/v1/webapi/saml/acs` where `teleport.example.com` is the
-public name of the teleport web proxy service:
+<Admonition type="tip">
+Set <Var name="teleport.example.com:443" description="Your Teleport proxy address and port, or cloud tenant." /> here with your Teleport Proxy address and port, or Teleport Cloud tenant (e.g. `company.teleport.sh:443`) to fill out the values below.
+</Admonition>
+
+- **Login URL**:
+  - `https://`<Var name="teleport.example.com:443"/>`/web/login`
```
[the remark this answers is not in the document]
Comment (alexfornuto) [reply docs/pages/access-controls/sso/one-login.mdx:49]: I found that I couldn't get this connector to work unless OneLogin was given the port as well.

File: examples/resources/onelogin-connector.yaml (hunk 1 above)
Comment (alexfornuto) [reply examples/resources/onelogin-connector.yaml:16]: This was a flub, that's my domain I was testing with, will scrub from the history.

File: docs/pages/access-controls/sso/one-login.mdx [hunk 3]
```diff
@@ -1,52 +1,59 @@
 ---
-title: SSH Authentication with One Login as an SSO provider
-description: How to configure SSH access using One Login as an SSO provider
-h1: SSH Authentication with OneLogin
+title: Teleport Authentication with One Login as an SSO provider
```
Comment (ptgott) [inline docs/pages/access-controls/sso/one-login.mdx:2]:
```suggestion
title: Teleport Authentication with OneLogin as an SSO Provider
```
- Using title case per the style guide
- Using the official product spelling

File: docs/pages/access-controls/sso/one-login.mdx [hunk 4]
```diff
@@ -1,52 +1,59 @@
...
+description: How to configure Teleport access using One Login as an SSO provider
```
Comment (ptgott) [inline docs/pages/access-controls/sso/one-login.mdx:3]: We should `s/One Login/OneLogin/` throughout

File: docs/pages/access-controls/sso/one-login.mdx [hunk 5]
```diff
@@ -1,52 +1,59 @@
...
+## Step 1/3. Create Teleport application in OneLogin
 
-Using OneLogin control panel, create a SAML 2.0 Web App in SAML configuration
-section:
+In the OneLogin control panel's main menu navigate to **Applications** ->
+**Add App**. Using the search box, select "SAML Custom Connector (SP Shibboleth)":
 
 ![SAML Custom Connector (SP Shibboleth)](../../../img/sso/onelogin/onelogin-saml-1.png)
 
+Define the new Application:
```
Comment (ptgott) [inline docs/pages/access-controls/sso/one-login.mdx:33]:
```suggestion
Define the new application:
```
This shouldn't be a proper noun here

File: docs/pages/access-controls/sso/one-login.mdx [hunk 6]
```diff
@@ -1,52 +1,59 @@
...
 - [Rectangular Icon](../../../img/sso/onelogin/teleportlogo@2x.png)
 
-Set `Audience`, `Recipient` and `ACS (Consumer) URL Validator` to the same value:
+From the application's **Configuration** page, set the following values:
 
-`https://teleport.example.com:3080/v1/webapi/saml/acs` where `teleport.example.com` is the
-public name of the teleport web proxy service:
+<Admonition type="tip">
+Set <Var name="teleport.example.com:443" description="Your Teleport proxy address and port, or cloud tenant." /> here with your Teleport Proxy address and port, or Teleport Cloud tenant (e.g. `company.teleport.sh:443`) to fill out the values below.
+</Admonition>
```
Comment (ptgott) [inline docs/pages/access-controls/sso/one-login.mdx:46]:
```suggestion
Set <Var name="teleport.example.com:443" description="Your Teleport Proxy Service address and port, or cloud tenant." /> here with your Teleport Proxy Service address and port, or Teleport Enterprise Cloud tenant (e.g. `company.teleport.sh:443`) to fill out the values below.
</Admonition>
```

File: docs/pages/includes/enterprise/samlauthentication.mdx [hunk 7]
```diff
@@ -1,57 +1,58 @@
...
-  </ScopedBlock>
+<ScopedBlock scope={["oss", "enterprise"]}>
+You can either edit the Teleport Auth Service configuration file or create a dynamic
+resource.
+</ScopedBlock>
 
-  <Tabs>
-  <TabItem scope={["cloud", "oss", "enterprise"]} label="Dynamic Resources (All Editions)">
+<Tabs>
+<TabItem scope={["cloud", "oss", "enterprise"]} label="Dynamic Resources (All Editions)">
```
Comment (ptgott) [inline docs/pages/includes/enterprise/samlauthentication.mdx:12]: I'm not sure if the scope switcher will work correctly if we have multiple tab items that share the same scope. Maybe remove the `oss` and `enterprise` scopes here? We'll also want to add a `team` scope to this tab item

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions

**Review by zmb3 — APPROVED** [review]

**Pull request merged**